pkg/dao/external_resource_test.go

```diff
@@ -0,0 +1,54 @@
+package dao
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type mockExternalResource struct {
+	mock.Mock
+}
+
+func (erd *mockExternalResource) ValidRepoMD(url string) (int, error) {
```

Comment:
Nitpick, but I think all the instances of "RepoMD" should be "Repomd"

Reply:
so "MD" is short for "Metadata".  So i'd think it'd be similar to URL, which we've capitalized, but maybe Md would make more sense?  "RepoMd"

Reply:
When I google it I see either "repomd" or "Repomd", which made me think it should be all lowercase. But if you want capital letter I think MD is fine.

Reply:
-1 to  "RepoMd", so I suggest just leave it as is.